src/pages/NewChatPage.js

```diff
@@ -233,50 +223,41 @@ class NewChatPage extends Component {
-                                        hideAdditionalOptionStates
-                                        forceTextUnreadStyle
-                                        shouldFocusOnSelectRow={this.props.isGroupChat}
-                                    />
-                                    {this.props.isGroupChat && lodashGet(this.state, 'selectedOptions', []).length > 0 && (
-                                        <FixedFooter>
-                                            <Button
-                                                success
-                                                onPress={this.createGroup}
-                                                style={[styles.w100]}
-                                                text={this.props.translate('newChatPage.createGroup')}
-                                            />
-                                        </FixedFooter>
-                                    )}
-                                </>
+                                <OptionsSelector
+                                    canSelectMultipleOptions={this.props.isGroupChat}
+                                    sections={sections}
+                                    selectedOptions={this.state.selectedOptions}
+                                    value={this.state.searchValue}
+                                    onSelectRow={option => (this.props.isGroupChat ? this.toggleOption(option) : this.createChat(option))}
+                                    onChangeText={(searchValue = '') => {
+                                        const {
+                                            recentReports,
+                                            personalDetails,
+                                            userToInvite,
+                                        } = OptionsListUtils.getNewChatOptions(
+                                            this.props.reports,
+                                            this.props.personalDetails,
+                                            this.props.betas,
+                                            searchValue,
+                                            [],
+                                            this.props.isGroupChat ? this.excludedGroupEmails : [],
+                                        );
+                                        this.setState({
+                                            searchValue,
+                                            userToInvite,
+                                            recentReports,
+                                            personalDetails,
+                                        });
+                                    }}
+                                    headerMessage={headerMessage}
+                                    hideAdditionalOptionStates
+                                    forceTextUnreadStyle
+                                    shouldFocusOnSelectRow={this.props.isGroupChat}
```

Comment:
This causes a minor [regression](https://github.com/Expensify/App/issues/20636) on mobile web, showing the keyboard for every contact selection. Handled it by excluding for mobile browsers